cypress/integration/basic-functionalities/local-tests/admin/application/1_Create_an_application.spec.js

```diff
@@ -0,0 +1,82 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+const email = Cypress.env('user').email
+const password = Cypress.env('user').password
+
+// Before running this test first sign up
+describe('Test for creating a role', () => {
+  before(() => {
+    cy.login({ email, password })
+  })
+
+  context('Test for creating an application without a name entered', () => {
+    it('should not be able to create an application', () => {
+      // This test might fail sometimes with an uncaught:error exception
+      // so just try to rerun the test or increase the wait time below
+      cy.wait(3000) // We wait for 3s in order the page to be fully loaded/rendered
+      cy.get('.nav-sidebar').contains('Applications').click()
+      cy.get('[data-test-id="button-new-application"]').click()
+      cy.get('[data-test-id="card-application-info"]').within(() => {
+        cy.get('[data-test-id="button-submit"].disabled').should('exist')
+      })
+    })
+  })
+  
+  context('Test for checking that delete and new buttons are not displayed when in create mode', () => {
+    it('should not be displayed when creating an application', () => {
+      cy.get('[data-test-id="button-new-application"]').should('not.exist')
+      cy.get('[data-test-id="card-application-info"]').within(() => {
+        cy.get('[data-test-id="button-delete"]').should('not.exist')
+      })
+    })
+  })
+
+  context('Test for checking that enabled settings is present in create mode', () => {
+    it('should be displayed when creating an application', () => {
+      cy.get('[data-test-id="card-application-info"]').within(() => {
+        cy.get('[data-test-id="checkbox-enabled"]').should('exist')
```

Comment:
Let's remove this test and mark enabled as true before creating a template. Afterwards, check if the change was persisted.